test/quantization/test_quant_api.py

```diff
@@ -196,7 +178,7 @@ def test_dynamic_quant_gpu_singleline(self):
     @unittest.skip("skipping for now due to torch.compile error")
     def test_dynamic_quant_gpu_unified_api_unified_impl(self):
         quantizer = XNNPackDynamicQuantizer()
-        m = ToyLinearModel().eval()
+        m = ToyTwoLinearModel(64, 32, 64).eval()
```

Comment:
it might be better to explicitly spell out everything (dtype=.., device=...) so there is no surprises I feel, can you update the rest of the PR to do that? make sure they match with the state before refactor